app/main.py

```diff
@@ -1,17 +1,42 @@
+import os
 from fastapi import FastAPI
 from fastapi.middleware.cors import CORSMiddleware
 from pydantic import BaseModel
+from dotenv import load_dotenv
 
 from .ml_model import predict_sentences
-from .utils import load_model, load_tokenizer
+from .utils import (
+    download_blob_from_bucket, 
+    load_file, 
+    load_model,
+    load_tokenizer
+)
+
+
+load_dotenv()
+GOOGLE_APP_CREDENTIALS = os.environ.get("GOOGLE_APPLICATION_CREDENTIALS")
+ML_STORAGE_BUCKET = "siloka-ml-resources"
+
 
 class MessagesReq(BaseModel):
     messages: str
 
 app = FastAPI()
 
+# Download latest ml resources
+download_blob_from_bucket(ML_STORAGE_BUCKET, "chatbot-model.h5", "model/chatbot-model.h5")
+download_blob_from_bucket(ML_STORAGE_BUCKET, "tokenizer.pickle", "model/tokenizer.pickle")
+download_blob_from_bucket(ML_STORAGE_BUCKET, "intents.json", "model/intents.json")
+
 model = load_model('./model/chatbot-model.h5')
 tokenizer = load_tokenizer('./model/tokenizer.pickle')
+intents = load_file('./model/intents.json')
```

Comment:
I think it would be better if you load ML resources before the server starts, to make sure everything is ready before it receive any requests